src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/PaymentBuilderTable/PaymentBuilderTable.tsx

```diff
@@ -50,14 +51,18 @@ const useGetPaymentBuilderColumns = ({
     [blockTime, finalizedTimestamp, slots],
   );
 
+  const { loading: isColonyContributorDataLoading } = useMemberContext();
```

Comment:
I actually think the solution you've implemented in `UserInfoPopover` fetches data from the same source, so we don't need to pass `isLoading` to `UserPopover`.
Maybe we could try removing that property. We can also try and see what happens if we revert the loading flag changes here (maybe it won't work though)

Reply:
@bassgeta it's used to pass the loading state of the data from the table, so I think this prop is also needed so that skeleton loader will be shown for all of the table items simultaneously. I also want to show the loading state on other items from the table when user data is being fetched.

Reply:
@bassgeta sorry, but I don't understand. The task is to add consistent styles to the table when any of its data is being loaded. There was some inconsistency here because `UserInfoPopover` had the loading state added inside and the skeleton was shown a bit longer than for other columns. Right now I'm getting information about two things - one about loading action data (via `isLoading` arg) and the second one about loading contributor data (`isColonyContributorDataLoading`). I know that `isColonyContributorLoading` is already checked in `UserInfoPopover` and that's why I'm passing there only the loading state of the action data.

Reply:
I am sorry, I should have been more thorough in my comment.
Basically the changes done here introduce a short circuit in `UserPopover` and `UserInfoPopover` to display a skeleton when a passed `isLoading` prop is `true`. At the same time, the component will still fetch the contributor data, even if you just want to display that skeleton.
While we are now conveniently displaying the loading state, since we are looking at the same loading flags, it may be abused in the future and lead to faulty behaviour. 
And generally if you want a different component rendered from a child, you don't introduce a short circuit, that's not a good pattern.
If you want the parent to render a loader instead of the child component, it should render the loader itself, not pass an `isLoading` prop to an uncontrolled component that fetches its own data.

Again, this issue stems from the fact that our `UserInfoPopover` component does way too much, but this change adds to its tech debt. If you really feel like not doing this, just add a @TODO comment to remove the flag and where it is used, and we will sort it when refactoring this component :)